yarn-project/circuits.js/src/structs/kernel/combined_constant_data.ts

```diff
@@ -44,6 +44,10 @@ export class CombinedConstantData {
     );
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +15]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->BTW could you check if this header is actually used? I recall Sean removed _some_ header in https://github.com/AztecProtocol/aztec-packages/pull/8990<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/9095?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
This is used in the public kernel. Leila is working on flattening our VM circuit public inputs struct so it doesn't reuse kernel structs that have fields in them unused by the VM. That being said, this header _might_ be where the AVM gets its tree roots from?